task.sh

```
@@ -1 +1,27 @@
 #! /bin/bash
+
```

Add strict bash safety flags near the top so the script fails fast and is safe for cron runs. For example: `set -euo pipefail`. This ensures the script exits on errors (requirement 3.7).